fotogalleri/gallery/templates/components/folder.html

```diff
@@ -7,7 +7,8 @@
   data-url="/view/{{ folder.full_path }}"
   data-id="{{ folder.pk }}"
 >
-  <i class="fas fa-folder" aria-hidden="true"></i>
-  <input type="checkbox" style="display: none;" />
-  <div class="folder-text">{{ folder.path }}</div>
+{{ folder.path }}
+  <input type="checkbox" style="display: none; margin-left: 10px;" />
+  <!-- <i class="fas fa-angle-right" aria-hidden="true" style="font-size: larger;"></i> -->
```

Comment:
Although `WIP`: should probably remove comment

Reply:
fixed